### PR TITLE
Add MLKit to FIRLogger constants.

### PR DESCRIPTION
**Pull request by ryanwilson**

### CODE REVIEW

File: Firebase/Core/Private/FIRLogger.h [hunk 1]
```diff
@@ -36,6 +36,7 @@ extern FIRLoggerService kFIRLoggerDynamicLinks;
 extern FIRLoggerService kFIRLoggerFirestore;
 extern FIRLoggerService kFIRLoggerInstanceID;
 extern FIRLoggerService kFIRLoggerInvites;
+extern FIRLoggerService kFIRLoggerMLKit;
```
Comment (paulb777) [inline Firebase/Core/Private/FIRLogger.h]: Sort the same as the others

File: Firebase/Core/FIRLogger.m [hunk 2]
```diff
@@ -37,6 +37,7 @@
 FIRLoggerService kFIRLoggerInstanceID = @"[Firebase/InstanceID]";
 FIRLoggerService kFIRLoggerInvites = @"[Firebase/Invites]";
 FIRLoggerService kFIRLoggerMessaging = @"[Firebase/Messaging]";
```
Comment (temrich) [inline Firebase/Core/FIRLogger.m]: nit: flip to match ordering in FIRLogger.h?

Comment (ryanwilson) [reply Firebase/Core/FIRLogger.m]: Whoops, thanks! `clang-format` swapped it on me :)

**Comment by paulb777** [comment]
Since MLKit is being shipped as multiple pods, do we want developers to have insight into which pod is which and which one in particular is sending the log message?

**Comment by davidair** [comment]
I think the naming is fine, AdMob is not the name of the pod either.

**Pull request merged**